### ISSUE TITLE
yubihsm-auth doesn't support yubikey 5 NFC FIPS

### ISSUE THREAD

**Issue by cnburke** [Yubico/yubihsm-shell#195]
*Plug in YubiKey 5 NFC FIPS*

user@ubuntu1804:~/Downloads$ sudo yubihsm-auth -a reset --verbose=10
trying to connect to reader 'Yubico Yubikey 4 OTP+U2F+CCID 00 00'
> 00 a4 04 00 07 a0 00 00 05 27 21 07 
< 6a 82 
Failed selecting application: 6a82
error: no usable reader found
Unable to connect: Error in PCSC call

*Unplug in YubiKey 5 NFC FIPS*
*Plug in YubiKey 5 NFC Non-FIPS*

user@ubuntu1804:~/Downloads$ sudo yubihsm-auth -a reset --verbose=10
trying to connect to reader 'Yubico Yubikey 4 OTP+U2F+CCID 00 00'
> 00 a4 04 00 07 a0 00 00 05 27 21 07 
< 79 03 05 04 03 90 00 
> 00 06 de ad 00 
< 90 00 
Device successfully reset

**Comment by cnburke** [comment]
Do you plan on implementing yubihsm-auth compatibility to the FIPS line of yubikeys? It doesn't make sense to me why there should be a difference in the two products...

**Comment by a-dma** [comment]
Yes, there are plans. As you can imagine some of those processes take extra time.
